### ISSUE TITLE
feat: mvencode

### ISSUE THREAD

**Issue by gergelyattilakiss** [codedthinking/Kezdi.jl#176]

**Comment by korenmiklos** [comment]
Can be done with `Missings.replace`: https://github.com/JuliaData/Missings.jl

**Comment by gergelyattilakiss** [comment]
It does not seem to be difficult at first, but we have to find a solution where we can replace more than 1 col at a time. I will check now Missings than.

**Comment by gergelyattilakiss** [comment]
With  little bit tweaking on code from @tpapp 
```
using DataFrames, Missings
df = DataFrame(a = [1, missing, 2], b = [missing, 3, 4])
for col in [:a,:b]
     df[!,col] = collect(Missings.replace(df[!,col],0))
end
```

**Comment by gergelyattilakiss** [comment]
Since from the command call we have a list of columns to be replaced and also a value to use instead of missings this should work.

**Comment by gergelyattilakiss** [comment]
fixed in #189
